Documentation/common-problems.md

```diff
@@ -23,25 +26,149 @@ One of the first things that should be done is to start the [rook-tools pod](./t
 * ceph osd tree
 * ceph pg stat
 
-
 Of particular note, the first two status commands provide the overall cluster health. The normal state for cluster operations is HEALTH_OK, but will still function when the state is in a HEALTH_WARN state. If you are in a WARN state, then the cluster is in a condition that it may enter the HEALTH_ERROR state at which point *all* disk I/O operations are halted. If a HEALTH_WARN state is observed, then one should take action to prevent the cluster from halting when it enters the HEALTH_ERROR state.
 
 There is literally a ton of Ceph sub-commands to look at and manipulate Ceph objects. Well beyond the scope of a few troubleshooting techniques and there are other sites and documentation sets that deal more with assisting one with troubleshooting a Ceph environment. In addition, there are other helpful hints and some best practices concerning a Ceph environment located in the [Advanced Configuration section](advanced-configuration.md). Of particular note there are scripts for collecting logs and gathering OSD information there.
 
 
-# Common Problems
+# Pod Using Rook Storage Is Not Running
+
+## Symptoms
+* The pod that is configured to use Rook storage is stuck in the `ContainerCreating` status
+* `kubectl describe pod` for the pod mentions one or more of the following:
+  * `PersistentVolumeClaim is not bound`
+  * `timeout expired waiting for volumes to attach/mount`
+* `kubectl -n rook-system get pod` shows the rook-agent pods in a `CrashLoopBackOff` status
+
+## Possible Solutions Summary
+* `rook-agent` pod is in a `CrashLoopBackOff` status because it cannot deploy its driver on a read-only filesystem: [Flexvolume configuration pre-reqs](./k8s-pre-reqs.md#flexvolume-configuration)
+* You are using Kubernetes 1.7.x or earlier and the Kubelet has not been restarted after `rook-agent` is in the `Running` status: [Restart Kubelet](#kubelet-restart)
```

Comment:
i would also add, if you are using 1.6.x, ensure to pass `--enable-controller-attach-detach=false` to kubelet

Reply:
OK, i will add a summary and link to that section.  has anyone hit this in the wild yet?

Reply:
@aramisjohnson did.